src/api/Feed.js

```diff
@@ -113,6 +113,23 @@ class Feed extends Base {
         this.taskLinksAPI = [];
     }
 
+    addFeedItem(id: string, item: FeedItem) {
+        if (!id || !item) {
```

Comment:
The flow types indicate that both of these are required parameters. Is this check needed?